src/theme/components/subnav.ts

```diff
@@ -0,0 +1,207 @@
+                  ? `${backgroundColor} `
+                  : "ui.white",
+            },
+          },
+        },
+      };
+      const secondaryActionsStyles = {
+        ...commonStyles(),
+        color: highlightOrLinkColor,
+        svg: {
+          fill: highlightOrLinkColor,
+          margin: { base: "0", md: null },
+          _dark: {
+            fill: backgroundColor
+              ? `${backgroundColor} !important`
+              : "dark.ui.link.primary-05 !important",
+          },
+        },
+        _hover: {
+          background: finalBackgroundColor,
+          svg: {
+            fill: highlightOrLinkColor,
+            _dark: {
+              fill: backgroundColor
+                ? `${backgroundColor}`
+                : "dark.ui.link.primary-05",
+            },
+          },
+        },
+      };
+      return {
+        base: {
+          ".selectedItem": {
+            color: highlightOrLinkColor,
+            fontWeight: "bold",
+            backgroundColor: finalBackgroundColor,
+            "&:hover": {
+              color: highlightOrLinkColor,
+            },
+          },
+          borderBottom: "1px solid",
+          borderColor: highlightOrBorderColor,
+          display: "flex",
+          justifyContent: "center",
+        },
+        container: {
+          maxWidth: "1280px",
+          // px: "s",
+          px: { base: "0", md: "xs" },
+          width: "100%",
+        },
+        listCleanup: {
+          // p: "xs",
+          p: { base: "s", md: "xs" },
+          gap: "xs",
+          li: {
+            marginEnd: "unset",
+          },
+        },
+        scrollableList: {
+          display: "flex",
+          overflowX: "auto",
+          whiteSpace: "nowrap",
+          position: "relative",
+          scrollbarWidth: "none",
+        },
+        primaryActions: {
+          width: "100%",
+          button: {
+            color: defaultLabelColor,
+            ...primaryActionsStyles,
+          },
+          a: {
+            color: `${highlightOrDefaultColor} !important`,
+            ...primaryActionsStyles,
+            svg: {
+              fill: `${highlightOrDefaultColor} !important`,
+              margin: { base: "0", md: null },
+              _dark: {
+                fill: "ui.white !important",
+              },
+            },
+          },
+        },
+        secondaryActions: {
+          width: "fit-content",
+          whiteSpace: "nowrap",
+          button: secondaryActionsStyles,
+          a: secondaryActionsStyles,
+        },
+        fadeEffect: {
+          position: "absolute",
+          top: 0,
+          right: 0,
+          height: "100%",
+          width: "50px",
+          background:
+            "linear-gradient(to left, rgba(255, 255, 255, 1) 0%, rgba(255, 255, 255, 0) 100%)",
+          pointerEvents: "none",
+          zIndex: 1,
+        },
+        ul: {
+          margin: "0 !important",
+        },
+        primaryList: {
```

Comment:
I don't think you're using this.

Reply:
Never mind, you are using it but I commented before you made your latest updates.